ChangeLog.md

```diff
@@ -25,11 +25,39 @@ See docs/process.md for more on how version tagging works.
   wasm/JS interface.  These function are no longer exported in all cases.  If
   your code directly calls these functions from JS, you can add them to
   `-sEXPORTED_RUNTIME_METHODS`.
+- The `LEGACY_RUNTIME` setting is no longer enabled by default.  If you use any
+  of these legacy runtime functions (except for library code with explict
+  dependencies) then you would need set `LEGACY_RUNTIME` on the command line or
```

Comment:
```suggestion
  dependencies) then you would need set `LEGACY_RUNTIME` on the command line, or
```
(makes it slightly clearer it is either that one thing, or all the rest of the sentence)